org.openstenoproject.Plover.yaml

```
@@ -0,0 +1,46 @@
+app-id: org.openstenoproject.Plover
+runtime: org.kde.Platform
+runtime-version: "6.9"
+sdk: org.kde.Sdk
+base: io.qt.PySide.BaseApp
+base-version: "6.9"
+command: plover
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --device=all
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+rename-desktop-file: plover
```

```suggestion
rename-desktop-file: plover.desktop
```

it is the **file name**